appveyor.yml

```diff
@@ -80,25 +79,31 @@ test_script:
             | Select-String -Pattern '<test-run id="[^"]*"\s+testcasecount="(\d+)"\s+result="([^"]*)"\s+total="(\d+)"\s+passed="(\d+)"\s+failed="(\d+)".*' `
             | foreach {
                 $result = $_.Matches.Groups
-                #$result[0].Value
+
                 $total  = $result[1].Value
                 $passed = $result[4].Value
                 $failed = $result[5].Value
                 $notPassed = $total - $passed
                 $testErrors = $failed
                 $testName = $resultFilePath.Split('\\')[-3]
                 $artifactName = "TestResult#${testRun}.${testErrors}err.${testName}"
-                Push-AppveyorArtifact $resultFilePath -FileName "${artifactName}.xml"
+
+                Write-Host "Pass#${testRun}: passed: $passed, failed: $failed, not passed: $notPassed, total: $total"
+
                 if ($notPassed -ne 0) {
                     $logFile = Get-ChildItem -recurse artifacts\$testname*.log
                     Get-Content $logFile
```

Comment:
This file is printed by the Arcade script now.
```suggestion
```

Reply:
Ah! Thank you, will clean up